### PR TITLE
Fixes lint:test and serve:test with the default files

### PR DESCRIPTION
**Pull request by tmorton**
I ran into problems running tests with a newly-generated app.  This PR fixes those issues.
- The `lint:test` task needs some options to pass the default mocha test file.
- The `serve:test` task needs a route to the bower components
- The `serve:test` task should open a browser like the other `serve` tasks.

### CODE REVIEW

File: app/templates/gulpfile.babel.js [hunk 1]
```diff
@@ -25,18 +25,28 @@ gulp.task('styles', () => {<% if (includeSass) { %>
...
   };
 }
+const testLintOptions = {
+  "env": {
+    "mocha": true
+  },
+  "globals": {
+    "assert": false,
+    "expect": false,
+    "should": false
```
Comment (kevva) [inline app/templates/gulpfile.babel.js]: Remove `"` from the keys.

File: app/templates/gulpfile.babel.js [hunk 2]
```diff
@@ -123,11 +133,13 @@ gulp.task('serve:dist', () => {
 gulp.task('serve:test', () => {
   browserSync({
     notify: false,
-    open: false,
```
Comment (kevva) [inline app/templates/gulpfile.babel.js]: Why?

**Comment by tmorton** [comment]
I'm not sure why Travis is failing.  It's only failing the Node 0.10 check, which passes on my machine.

**Comment by silvenon** [comment]
Maybe better to have those options in the `package.json` where the rest of them are? What do you think @kevva?

**Comment by silvenon** [comment]
Btw, I restarted the test and now it passes. It sometimes times out, we should probably increase the timeout.

**Comment by tmorton** [comment]
@silvenon If they're in `package.json` won't they apply to the regular `lint` task as well?  That seems undesirable.

**Comment by silvenon** [comment]
You're right, it's best not to mix them.

**Comment by silvenon** [comment]
LGTM, thanks! :+1:

**Pull request merged**